test/presenters/statistics_announcement_presenter_test.rb

```diff
@@ -64,6 +64,14 @@ def schema_name
     refute statistics_announcement_national.release_date_changed?
   end
 
+  test 'an announcement is forthcoming if it is not cancelled' do
+    assert presented_item('official_statistics').forthcoming_publication?
```

Comment:
the 'official_statistics' presented item is wrapped in a method on line 91. We should use that.

Reply:
👍  fixed in 2ba1720